pandas/tests/api/test_api.py

```diff
@@ -51,7 +51,7 @@ class TestPDApi(Base):
                ]
 
     # these are already deprecated; awaiting removal
-    deprecated_classes = ['Panel']
+    deprecated_classes = ['']
```

Comment:
should this be an empty list?

Reply:
Yep

Reply:
> should this be an empty list?

This has not been updated yet?

Reply:
It was but I force pushed last without pulling this locally. Give me a sec